odata-protocol/11.4 Data Modification.md

```diff
@@ -1287,6 +1299,9 @@ If an individual change fails due to a failed dependency, it MUST be
 annotated with the term [`Core.DataModificationException`]($$$OData-VocCore$$$#DataModificationException) and SHOULD specify
 a `responseCode` of `424` ([Failed Dependency](#ResponseCode424FailedDependency)).
 
+Regardless of the `continue-on-error` preference, the collection update must happen
```

Comment:
Continue-on-error necessarily means non-atomic.

Reply:
In the absence of continue-on-error, the operation must complete in an atomic manner.